CS342MemberViews/Sources/JenMoore/GalleryView.swift

```diff
@@ -0,0 +1,97 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+public struct GalleryView: View {
+    @Binding var accentColor: Color
+    @State private var detail: String?
+    @State var showModal = false
+    
+    private let imageNames = ["NewYork", "Riggy", "Stanford"]
```

Comment:
Great job with making these constants and rendering your view based on these collections!